README.md

```diff
@@ -97,3 +97,28 @@ Contributing to this repository requires:
 1. to make a Pull Request
 2. all code contributed must be pep8 compliant
 3. all unit tests must pass
+
+
+# Instructor notes: tutorial flow
+
+1. Introduction 20 mins
+    - Purpose and goals of the tutorial
+    - What is ETS? Why pick it to build tools? When to pick it to build tools?
+2. Starting script (10 min)
+3. Traits (30 mins)
+  - 
+  - Simple exercise (10 mins)
+4. Basic TraitsUI (35 mins) 
+  - Introduction to TraitsUI views
+  - The MVC paradigm
+  - TraitsUI exercise (20 mins)
```

Comment:
This is a best guess for now, they're bound to change as I think about how to present this best.